### PR TITLE
Feature/data split

### PR DESCRIPTION
**Pull request by gvechini**
Adicionado a componente Data-Split, experiment, deployment, test, e confg.
Componentes de classificação e regressão atualizadas para poderem:

1. Usar a componente split-data, aqui precisei mexer nos metadados;
2. Terem nova opção de poderem usar o RandomizedSearchCV para encontrar os melhores hiperparâmetros daquele modelo.

### CODE REVIEW

**Comment by lborro** [comment]
@gvechini 
Os casos tratados no componentes de split me parecem OK. Cobrem grande parte das necessidades referentes à otimização de hiperparâmetros, bem como para estimar desempenho dos modelos.

Talvez valha a pena adicionar mais comentários nas células. Também seria legal descrever os parâmetros entrada. Por exemplo, explicar o porquê da estratificação, dar alguns exemplos de uso. Você pode até se basear nos tutoriais do sklearn.

**Comment by lborro** [comment]
@gvechini 
Só resolver conflito no arquivo `config.json`, que eu faço a aprovação do PR. Você poderia fazer isso?

**Comment by gvechini** [comment]
@lborro Mudei a descrição do patametro strat:
De:
->strat = None #@param {type:"feature",multiple:false,label:"Atributo para fazer o balanceamento.", description: " Será usado a codificação ordinal se o atributo for categórico."}

Para
-> strat = None #@param {type:"feature",multiple:false,label:"Garante a mesma proporção de cada categoria da feature selecionada para os conjuntos de treino, validação e teste.", description: "Será usado a codificação ordinal se o atributo for categórico."}

Creio que o conflito no config.json foi resolvida agora.

Adicionei a ultilização do data-split em 3 componentes de engenharia de atributos, havia esquecido.
